deployment/debug_echo_configuration.yaml

```diff
@@ -0,0 +1,14 @@
+# Source: akri/templates/debug-echo-configuration.yaml
+apiVersion: akri.sh/v0
```

Comment:
Instead of this file, could you install the configuration with Helm:
```sh
helm install debug-config akri-helm-charts/akri \
 --set controller.enabled=false \
 --set agent.enabled=false \
 --set rbac.enabled=false \
 --set debugEcho.configuration.enabled=true
```

Reply:
Oh, nice! That would be better, thanks Kate!

Reply:
Should this file be deleted if using helm now?

Reply:
Yes, will delete the now unused files.